code/modules/projectiles/projectile/energy/nuclear_particle.dm

```diff
@@ -30,7 +30,7 @@
 	P.fire(angle)
 
 /obj/item/projectile/energy/nuclear_particle/wimpy
-	irradiate = 100
+	irradiate = 500
```

Comment:
no

Reply:
it only affects mob not surrounding

Reply:
so if it hits an object, it only damaging it not spreading radiation but if it hits a mob their its radiation will increase

Reply:
you know what im just gonna use the base nuclear